Makefile

```diff
@@ -4,6 +4,13 @@ FAKTORY_PORT=7419
 FAKTORY_PORT_SECURE=17419
 FAKTORY_PORT_UI=7420
 
+.PHONT: precommit
+precommit: fmt check test/doc test/e2e test/e2e/tls
+
+.PHONY: fmt
+fmt:
+	cargo fmt
```

Comment:
@jonhoo I remember you did not like this entry when we were first adding the make file, saying that we all have our formatting set up. I hope though that this command - when issued - will not interfere with any local set up, wdyt?

Reply:
It's fine to have it here I suppose, it's more that I would expect people to generally have format-on-write turned on for Rust in their editors.